src/styles/index.css

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
+}
+
+.product-card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+}
+
+.product-card__image {
+  background: center url(../images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin: 32 19px;
```

Comment:
The 'margin' property should have four values for top, right, bottom, and left. Currently, it has only two values, which will apply to vertical and horizontal margins respectively. If you intended to set all four sides, please provide four values.